src/js/azdata/azdataQueryPlan.js

```diff
@@ -73,6 +84,63 @@ azdataQueryPlan.prototype.init = function(container, iconPaths)
         icons.push(iconName);
     }
 
+    ////////////////////////////////////////////////////////////////////////////////////
+    // temporary button logic to confirm zoom in and zoom out works.
+    
+    var buttons = document.createElement('div');
+	buttons.style.position = 'absolute';
+	buttons.style.overflow = 'visible';
+
+    var bs = graph.getBorderSizes();
+    buttons.style.top = (container.offsetTop + bs.y) + 'px';
+	buttons.style.left = (container.offsetLeft + bs.x) + 'px';
+    
+    var left = 0;
+	var bw = 16;
+	var bh = 16;
+
+    function addButton(label, eventCallback)
```

Comment:
I thought the plan was to expose the APIs to do zooming in the control.  But to have the actual Zoom In/Zoom Out buttons created in the ADS query plan pane toolbar.  Is that not what we planned?